src/patterns/ec2-app.ts

```diff
@@ -192,19 +194,20 @@ export class GuEc2App {
 
     const listener = new GuHttpsApplicationListener(scope, "Listener", {
       app,
+      loadBalancer,
+      certificate,
+      targetGroup,
       // When open=true, AWS will create a security group which allows all inbound traffic over HTTPS
```

Comment:
Correcting my own comment based on my research this morning!

```suggestion
      // When open=true, AWS will create an ingress rule which allows all inbound traffic over HTTPS
```